apps/gateway/package.json

```diff
@@ -28,7 +28,8 @@
     "hono": "^4.5.3",
     "jet-paths": "^1.0.6",
     "lodash-es": "^4.17.21",
-    "zod": "^3.23.8"
+    "zod": "^3.23.8",
+    "rate-limiter-flexible": "^5.0.3"
```

Comment:
hmm not sure how you installed these deps but you are missing the changes to pnpm lock file. Make sure to install any deps with `pnpm install`